documentation/book/cluster-operator.adoc

```diff
@@ -1060,111 +1060,124 @@ They will not be added to the Docker image from a previous build.
+  - When the rack feature is not used no binding is created.
+* The Zookeeper pods use the default `ServiceAccount` as they have no need to access {ProductPlatformName} resources
+* The Topic Operator pod uses a `ServiceAccount` called `<cluster-name>-topic-operator` (where `<cluster-name>` is a placeholder for the name of the `Kafka` resource)
+    - The Topic Operator produces {ProductPlatformName} events with status information, so the `ServiceAccount` is bound to a `ClusterRole` called `strimzi-topic-operator` which grants this access via the `strimzi-topic-operator-role-binding` `RoleBinding`.
+
+The pods for `KafkaConnect` and `KafkaConnectS2I` resources use the default `ServiceAccount`, since they require no access to {ProductPlatformName} resources.
+
+
+==== Using a `ServiceAccount`
+
+The Cluster Operator is best run using a `ServiceAccount`:
 
 [source,yaml,options="nowrap"]
-.Example ServiceAccount for the Cluster Operator
+.Example `ServiceAccount` for the Cluster Operator
 ----
-include::examples/install/cluster-operator/01-service-account.yaml[]
+include::examples/install/cluster-operator/01-ServiceAccount-strimzi-cluster-operator.yaml[]
 ----
 
-The Deployment of the operator then needs to specify this in the `serviceAccountName` of its `template.spec`:
+The `Deployment` of the operator then needs to specify this in its `spec.template.spec.serviceAccountName`:
 
 [source,yaml,numbered,options="nowrap",highlight='12']
-.Partial example Deployment for the Cluster Operator
+.Partial example `Deployment` for the Cluster Operator
 ----
-include::examples/install/cluster-operator/08-deployment.yaml[lines=1..13]
+include::examples/install/cluster-operator/05-Deployment-strimzi-cluster-operator.yaml[lines=1..13]
       # ...
 ----
 
-Note line 12, where the the `strimzi-cluster-operator` ServiceAccount is specified as the `serviceAccountName`.
+Note line 12, where the the `strimzi-cluster-operator` `ServiceAccount` is specified as the `serviceAccountName`.
 
-If the rack awareness feature is used, a Kafka init container will be started before the broker container in order to enable it.
-The application inside this container is best run using a ServiceAccount:
 
-[source,yaml,options="nowrap"]
-.Example ServiceAccount for the Kafka init container
-----
-include::examples/install/cluster-operator/04-service-account-kafka.yaml[]
-----
+==== Defining `ClusterRoles`
 
-The Cluster Operator will be in charge of configuring this ServiceAccount as `serviceAccountName` for the Kafka init container during cluster deployment.
+The Cluster Operator needs to operate using `ClusterRoles` that give it access to the necessary resources. 
+Depending on the {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the `ClusterRoles`.
 
-==== Defining a Role
+NOTE: Cluster administrator rights are only needed for the creation of the `ClusterRoles`. 
+The Cluster Operator will not run under the cluster admin account.
```

Comment:
I think that if you have a line break here it won't render as part of the note.  Do either you do this:

`NOTE: Cluster administrator rights are only needed for the creation of the `ClusterRoles`.  The Cluster Operator will not run under the cluster admin account.`

Or:

```
[NOTE]
====
Cluster administrator rights are only needed for the creation of the `ClusterRoles`.  
The Cluster Operator will not run under the cluster admin account.
====
```

AsciiDoc is particular about line spacing.

Reply:
It renders like this:

![screenshot from 2018-07-20 14-12-07](https://user-images.githubusercontent.com/879487/43004206-f2ef3906-8c26-11e8-98a6-4b498431a5f8.png)

Reply:
Oh, good to know!